src/clusterfuzz/_internal/metrics/monitor.py

```diff
@@ -108,7 +108,12 @@ def run(self):
         if time_series:
           create_time_series(project_path, time_series)
       except Exception:
-        logs.log_error('Failed to flush metrics.')
+        if environment.is_android():
+          # FIXME: This exception is extremely common on Android. We are already
+          # aware of the problem, don't make more noise about it.
+          logs.log_warn('Failed to flush metrics.')
```

Comment:
Just making sure: will this log the exception and its stack trace too?

Reply:
It won't.

Reply:
That's too bad.